layouts/newsletter/single.html

```diff
@@ -2,9 +2,9 @@
 
 {{- $pdf := printf "newsletter_pdf/%s.pdf" .File.BaseFileName }}
 
-<article class="newsletter">
+<article>
```

Comment:
I am wondering why the class on the article was removed. 

I did look through the stylesheet and `newsletter` class doesn't seem to do much.  Is this no longer needed?